guides/common/modules/proc_enrolling-project-server-in-freeipa-domain.adoc

```diff
@@ -0,0 +1,116 @@
+[id="enrolling-project-server-in-freeipa-domain_{context}"]
+= Enrolling {ProjectServer} in a {FreeIPA} domain
+
+Create a host entry for your {ProjectServer} system in the {FreeIPA} LDAP and configure the system to be a client in your {FreeIPA} domain.
+
+.Prerequisites
+
+* An existing {FreeIPA} server.
```

Comment:
I'd specify that admin user access or user with certain privileges are necessary.  But there are many options so it could be something generic, for example:
* Admin role or privileges required to enroll new hosts.

Reply:
Added.